docs/contributing/orm_models.md

```diff
@@ -0,0 +1,458 @@
+# ORM Models
```

Comment:
This document is the updated "model layer" one, but git is tragically unable to see that it's just been renamed, because I've added too much new content.

As a side note, we really need to actually clean up the developer documentation, there's a ton of stuff that is stubbed out and probably has no place here (e.g. guides to uploading data)